binancial/data/get_klines_historical.py

```
@@ -1,10 +1,12 @@
-def get_klines_historical(symbol="BTCUSDT",
+def get_klines_historical(client,
```

Ruff (0.8.2)

Missing return type annotation for public function `get_klines_historical`

(ANN201)